Menu rearrangement

jsteinaker opened the pull request:
As discussed in #80

I did a make catkeys and added the resulting file to the commit, not sure if it's the right thing to do. How do we update that in Polyglot later, is it done automatically?

humdingerb commented:
I can update those catkeys at Polyglot.

humdingerb commented:
To make this change complete, you could also update the screenshot and ReadMe.

humdingerb commented on Source/MainWindow.cpp:
```diff
@@ -41,7 +41,13 @@ BMenuBar*
 {
 	BMenuBar* menubar = new BMenuBar("menu");
-	BMenu* menu = new BMenu(B_TRANSLATE("Edit"));
+	BMenu* menu = new BMenu(B_TRANSLATE("Weather"));
```
I think using B_TRANSLATE_SYSTEM_NAME("Weather") here would be correct and saves a duplicated 'effort' of translation.

jsteinaker replied:
Indeed, thank you. Updated with your other suggestions.

humdingerb commented:
Alright! How about that updated screenshot? :)

nexus6-haiku commented:
All good for me, @humdingerb can we merge it?

humdingerb commented:
I may have missed the updated screenshot before... Though, I would have preferred to keep its location "Hacker Valley, WV". :)
Anyhoo, merge away!

nexus6-haiku commented:
I agree, it looks more iconic! @jsteinaker could you please create a new screenshot?

jsteinaker commented:
Done.

Also, I've been working in giving Weather better scaling support for HiDPI displays, since most (if not all) of the values are hardcoded, and it's too small for 1920x1080 (let alone anything bigger). I've used the system's font size as a reference, do you think it's the right approach to do this or how is it done usually? Using the screen resolution?

While at it, I moved a couple of things around, changed some numbers and...

![weathernew](https://user-images.githubusercontent.com/10001037/195169819-2ad34476-2a91-455b-afcf-c109e86b1918.png)

What do you think about it?

(don't be scared, the screenshot for closing this pull request is a different one and it's already in place :) )

nexus6-haiku commented:
We will completely refresh the UI sooner or later. It is quite outdated to be honest.In the meantime, we may fix certain aspects and I would go ahead with your proposal. There are a couple of things I would change, though.I would lay out the four elements one on top of the others and vertically aligned. The icon first, immediately below the temperature with a slightly smaller font than your screenshot, then the condition and the location.The idea is to gather those four elements in a separate compact child view which can be replicated on the desktop and let the user toggle the section below with the forecast. The user then can just drag the compact view alone or the whole view with the forecast.Does it make sense?Il giorno 11 ott 2022, alle ore 20:28, Javier Steinaker ***@***.***> ha scritto:﻿
Done.
Also, I've been working in giving Weather better scaling support for HiDPI displays, since most (if not all) of the values are hardcoded, and it's too small for 1920x1080 (let alone anything bigger). I've used the system's font size as a reference, do you think it's the right approach to do this or how is it done usually? Using the screen resolution?
While at it, I moved a couple of things around, changed some numbers and...

What do you think about it?
(don't be scared, the screenshot for closing this pull request is a different one and it's already in place :) )

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because your review was requested.Message ID: ***@***.***>
